### ISSUE TITLE
NTR: Xanthurenic acid high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8817]
New term request Xanthurenic acid high in urine (segal_061220195539):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000121
- label: Elevated urinary anthurenic acid level
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: 
- def: The amount of anthurenic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8817
- synonyms:

**Comment by MickeySegal** [comment]
Is **anthurenic** a typo in the label and def?  All I see elsewhere is xanthurenic.

**Comment by pnrobinson** [comment]
fixed, thanks! Indeed the X was missing.
